client/CBitmapHandler.cpp

```diff
@@ -42,33 +42,36 @@ SDL_Surface * BitmapHandler::loadH3PCX(ui8 * pcx, size_t size)
 	SDL_Surface * ret;
 
 	Epcxformat format;
-	int it=0;
-
-	ui32 fSize = read_le_u32(pcx + it); it+=4;

```

Comment:
This is example of general case when sometimes multi statements may have their use, but yep...
I guess i am getting too paranoid about theoretical cases while our codebase has almost 0 legit usage of "multi statements in one line"